common/src/main/java/com/microsoft/identity/common/internal/dto/RefreshToken.java

```diff
@@ -135,62 +122,45 @@ public void setFamilyId(final String familyId) {
         mFamilyId = familyId;
     }
 
-    /**
-     * Gets the username.
-     *
-     * @return The username to get.
-     */
-    public String getUsername() {
-        return mUsername;
-    }
-
-    /**
-     * Sets the username.
-     *
-     * @param username The username to set.
-     */
-    public void setUsername(final String username) {
-        mUsername = username;
-    }
-
+    //CHECKSTYLE:OFF
+    @SuppressWarnings("PMD")
     @Override
     public boolean equals(Object o) {
-        if (this == o) {
-            return true;
-        }
-
-        if (o == null || getClass() != o.getClass()) {
-            return false;
-        }
-
-        if (!super.equals(o)) {
-            return false;
-        }
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        if (!super.equals(o)) return false;
 
         RefreshToken that = (RefreshToken) o;
 
-        if (mClientInfo != null ? !mClientInfo.equals(that.mClientInfo) : that.mClientInfo != null) {
+        if (mClientInfo != null ? !mClientInfo.equals(that.mClientInfo) : that.mClientInfo != null)
             return false;
-        }
-
-        if (mFamilyId != null ? !mFamilyId.equals(that.mFamilyId) : that.mFamilyId != null) {
+        if (mFamilyId != null ? !mFamilyId.equals(that.mFamilyId) : that.mFamilyId != null)
             return false;
-        }
-
-        if (mTarget != null ? !mTarget.equals(that.mTarget) : that.mTarget != null) {
-            return false;
-        }
-
-        return mUsername != null ? mUsername.equals(that.mUsername) : that.mUsername == null;
+        return mTarget != null ? mTarget.equals(that.mTarget) : that.mTarget == null;
     }
+    //CHECKSTYLE:ON
 
+    //CHECKSTYLE:OFF
+    @SuppressWarnings("PMD")
     @Override
     public int hashCode() {
         int result = super.hashCode();
-        result = UNIQUE_ID_LENGTH * result + (mClientInfo != null ? mClientInfo.hashCode() : 0);
-        result = UNIQUE_ID_LENGTH * result + (mFamilyId != null ? mFamilyId.hashCode() : 0);
-        result = UNIQUE_ID_LENGTH * result + (mTarget != null ? mTarget.hashCode() : 0);
-        result = UNIQUE_ID_LENGTH * result + (mUsername != null ? mUsername.hashCode() : 0);
+        result = 31 * result + (mClientInfo != null ? mClientInfo.hashCode() : 0);
```

Comment:
add a comment here to say this method is generated.

Reply:
Resolved via 895329453073bcb92b7d91b9c4b7ae38a6f14c0d